.github/workflows/deploy.yml

```diff
@@ -144,76 +153,169 @@ jobs:
           # Save the output for writing to the task definiton
           echo "::set-output name=image::$IMAGE_NAME"
 
+      ##################################
+      ##################################
+      # Update website Task definition #
+      ##################################
+      ##################################
+
+      # We get the existing task definition to update
+      # Note that this is passed into the first update action
+      # but then chained from action to action for the others.
+      - name: Download website task definition
+        run: |
+          aws ecs describe-task-definition --task-definition ${{ env.website_task_definition_name }} --query taskDefinition > website-task-definition.json
+
+      - name: Update website task definition for puma container
+        id: update_definition_for_website_puma
+        uses: aws-actions/amazon-ecs-render-task-definition@v1
+        with:
+          task-definition: website-task-definition.json
+          container-name: puma
+          image: "${{ steps.build_rails.outputs.image }}"
+
+      - name: Update website task definition for nginx container
+        id: update_definition_for_website_nginx
```

Comment:
I believe this needs to be `${{ steps.update_definition_for_website_puma.outputs.task-definition }}`

The `website-task-definition.json` is immutable, so each step has to daisy-chain onto the previous one.